heroic-component/src/main/java/com/spotify/heroic/cluster/ClusterShard.java

```diff
@@ -77,10 +79,17 @@
             .retryUntilResolved(() -> {
                 final ClusterNode.Group next = it.next();
                 return function.apply(next).catchFailed(throwable -> {
+                    reporter.reportClusterNodeRPCError();
```

Comment:
Capitalization rules ignores acronyms, this method should be `reportClusterNodeRpcError()`. For the reasoning, consider the readability of HTTPXMLRPC vs HttpXmlRpc.